mobile/library/common/http/client.cc

```diff
@@ -450,14 +450,20 @@ void Client::DirectStreamCallbacks::latchError() {
   }
 
   error_msg_details.push_back(absl::StrCat("ERROR_CODE: ", error_->error_code));
+  std::vector<std::string> response_flags(info.responseFlags().size());
+  std::transform(info.responseFlags().begin(), info.responseFlags().end(), response_flags.begin(),
+                 [](StreamInfo::ResponseFlag flag) { return std::to_string(flag.value()); });
+  error_msg_details.push_back(absl::StrCat("RESPONSE_FLAGS: ", absl::StrJoin(response_flags, ",")));
```

Comment:
The idea behind response flags is that they map to the error code (e.g. `net.dns`, `net.connect`, etc).  Is the idea here that those are too course grained, so it's better if we know the response flags directly from the error details?

Reply:
agreed